starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/Shopping_List.java

```diff
@@ -0,0 +1,61 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by ds on 10/25/16.
+ */
+
+public class Shopping_List {
```

Comment:
Best practice is not to have underscores in class names, so just `ShoppingList` instead of `Shopping_List`. Also, this class represents a single shopping list item, not really a whole collection of items, so it would make more sense to call it something like `ShoppingListItem` or `ShoppingItem` so its clear that each object of this class is a single item and not a whole list.